LICENSES/README.md

```diff
@@ -0,0 +1,62 @@
+# systemd Project Licensing
+
+## Main License
+
+The systemd project uses single-line references to Unique Licese Identifiers as
+defined by the Linux Foundation's SPDX project (https://spdx.org/). The line in
+each individual source file identifies the license applicable to that file.
+
+The current set of valid, predefined SPDX identifiers can be found on the SPDX
+License List at https://spdx.org/licenses/.
+
+The 'LICENSES/' directory contains all the licenses used by the sources included in
+the systemd project source tree.
+
+Unless otherwise noted, the systemd project sources are licensed under the terms
+and conditions of the **GNU Lesser General Public License v2.1 or later**.
+
+New sources that cannot be distributed under LGPL-2.1-or-later will no longer
+be accepted for inclusion in the systemd project to maintain license uniformity.
+
+## Other Licenses
+
+The following exceptions apply:
+
+ * some udev sources under src/udev/ are licensed under **GPL-2.0-or-later**, so the
+   udev binaries as a whole are also distributed under **GPL-2.0-or-later**.
```

Comment:
i'd weaken this language too.

maybe: "…, so the udev binaries as a whole should probably be considered GPL-2.0-or-later" or so.

i.e. "should probably be considered" sounds better then "is" to me, given we are not lawyers

Reply:
as mentioned below this is clear, the compatibility of GPL and LGPL is one way, so if there's a mix it has to be GPL